core/homophones/homophones.csv

```diff
@@ -252,10 +262,12 @@ friar,fryer
 fur,fir
 gaffe,gaff
 gate,gait
+gator,gaiter
 gamble,gambol
 gel,jell
 gene,Jean
 jibe,gibe
+gui,gooey
```

Comment:
'gui' is not a word so would remove this; consider adding to the default abbreviation list instead.